tests/css_selector_list/selectors.css

```diff
@@ -71,3 +71,19 @@
   .child-one {}
   .child-two {}
 }
+
+.parent>>>.child {}
+.parent >>> .child {}
+.parent  >>>  .child {}
+.parent>>>{
+  .child-one {}
+  .child-two {}
+}
+.parent >>> {
+  .child-one {}
+  .child-two {}
+}
+.parent  >>>  {
+  .child-one {}
+  .child-two {}
+}
```

Comment:
Could you add a few similar tests where the `>>>` is before the `.child-x` selectors?

Reply:
@j-f1 yep, but it is invalid css, should we handle invalid css?

Reply:
Might be valid SCSS though

Reply:
@lydell right now `scss` don't support deep selector, also for `scss` valid only `.parent [combinator] { .child { color: red }}` structure